README.md

```diff
@@ -1,24 +1,73 @@
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Project Documentation 📄
+- [Live Demo]()
+- [API Documentation](http://127.0.0.1:5000/api-docs/index.html)
 
-Things you may want to cover:
+## Team members: 4 (Dozie Udeagha, SaharSaba Amiri, Mean Mohammad, Bright Kweku Ahiadeke )
 
-* Ruby version
+# Built With 🛠️
+- Programming Languages: HTML, CSS, JavaScript, Ruby.
+- Frameworks: React, Redux, Rails and Bootstrap.
+- GitHub Linters.
+- Code Editor: VS Code.
 
-* System dependencies
+## Kanban Board
+- Click this [link](https://github.com/users/maenkhraisha/projects/6) to view the Kanaban Board
+- Click this [link](https://docs.google.com/document/d/1GcBpwqr2NsF9umMA4qthxuBxnEjcLagS8p_tt5CwHPc/edit?usp=sharing) to view the initial state of the kanban board
+- Click [here](https://github.com/maenkhraisha/hotel-reservation) for the main branch of the repo
+# Getting Started
+To get a local copy up and running follow these simple steps.
 
-* Configuration
+### Prerequisites
+- A web browser like Google Chrome.
+- A code editor like Visual Studio Code with Git and nodejs.
+- You can check if Git is installed by running the following command in the terminal.
+- `$ git --version`
+- Clone the repo from here.
+- `$ git clone https://github.com/maenkhraisha/hotel-reservation.git`
 
-* Database creation
+# Usage
+## Run locally
+- Make your way to the correct directory by running this command:
+- `$ cd hotel-reservation`
+- Run npm install
+- Then you can run this command to configure the app in your PC:
+- `$ rails db:drop db:create db:migrate db:seed`
+- For starting the server to see the app in windows you can run this command:
+- `foreman start`
 
-* Database initialization
+# Authors
+## 👤 Mean Mohammad
+- [GitHub](https://github.com/maenkhraisha)
+- [Twitter](https://twitter.com/AlkhryshaM)
+- [Linkedin](https://www.linkedin.com/in/maen-al-khraisha/)
 
-* How to run the test suite
+## 👤 SaharSaba Amiri
+- [GitHub](https://github.com/Sahar-SE)
+- [Twitter](https://twitter.com/SaharSabaAmiri)
+- [Linkedin](https://www.linkedin.com/in/sahar-saba-amiri/)
 
-* Services (job queues, cache servers, search engines, etc.)
+## 👤 Bright Kweku Ahiadeke
+- [GitHub](https://github.com/kwekubright)
+- [Twitter](https://twitter.com/kwekubright_)
+- [Linkedin](https://www.linkedin.com/in/kwekubright)
 
-* Deployment instructions
+## 👤 Dozie Udeagha
+- [GitHub](https://github.com/udeaghad)
+- [Twitter](https://twitter.com/theodoz)
+- [Linkedin](https://www.linkedin.com/in/)
 
-* ...
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Microverse
+- Original design idea by [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign/modules/173005577).
+
+## 📝 License
+This project is MIT licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪